pymc3/gp/util.py

```diff
@@ -12,10 +12,13 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-from scipy.cluster.vq import kmeans
+import warnings
+
 import numpy as np
 import theano.tensor as tt
-import warnings
+import theano.tensor.slinalg
```

Comment:
you'll need to silence the pylint warning
```suggestion
import theano.tensor.slinalg  # pylint: disable=unused-import
```